### PR TITLE
slides for chapter 1 (made after the meeting)

### PR DESCRIPTION
**Pull request by russHyde**
Content for the slides on the first chapter (basically just an intro to ML) and the preface. Presented 7th May.

### CODE REVIEW

**Comment by russHyde** [comment]
@jonthegeek could you review this, please. I'll review any subsequent weeks (when they aren't presented by me).

**Comment by russHyde** [comment]
For some reason the actions aren't running

**Comment by jonthegeek** [comment]
> For some reason the actions aren't running

Since you haven't pred this one yet and I haven't added you as an editor, the checks needed approval. I'll give it a look in an hour or so. Thanks again!

File: 01-foundations.Rmd [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+**Learning objectives:**
+
+- Understand the purpose for the book
+- Know the names of any python packages relevant to the book
+- Have an overview of
+  - what machine learning is
+  - the different types of machine learning problem
+  - different types of uncertainty
+
+## Origin of the book
```
Comment (jonthegeek) [inline 01-foundations.Rmd]:
```suggestion
## Origin of the book {-}
```

File: 01-foundations.Rmd [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Hardware advances
+- GPUs
+
+Crowd sourcing data collection
+- Amazon Mechanical Turk
+
+Unifying lens for the book is “Probabilistic modeling and Bayesian decision theory”
+
+## Python packages
```
Comment (jonthegeek) [inline 01-foundations.Rmd]:
```suggestion
## Python packages {-}
```

File: 01-foundations.Rmd [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- JAX
+  - numerics on tensors and automatic differentiation
+- PyTorch
+  - tensor library for deep learning
+- TensorFlow
+  - framework for building ML pipelines (?)
+- PyMC
+  - probabilistic programming MCMC etc
+
+## Notebooks for the book
```
Comment (jonthegeek) [inline 01-foundations.Rmd]:
```suggestion
## Notebooks for the book {-}
```

File: 01-foundations.Rmd [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- What is machine learning from a probabilistic perspective?
+- Why take a probabilistic approach to ML?
+
+## Types of Machine Learning Problem {-}
+
+- supervised learning (classification, regression, )
+- unsupervised learning (clustering, latent variables)
+- reinforcement learning (learn how to interact with env)
+
+## Types of 'uncertainty'
```
Comment (jonthegeek) [inline 01-foundations.Rmd]:
```suggestion
## Types of 'uncertainty' {-}
```

**Review by jonthegeek — DISMISSED** [review]
I'll commit the code suggestions (adding {-} to section titles so they don't screw up the chapter list). I'll try to upgrade this one to quarto asap so you won't have to do that (and so you can more easily include Python code)!

**Pull request merged**